src/components/DataDictionary.tsx

```diff
@@ -76,11 +77,11 @@ export const columns: ColumnDescriptor[] = [
     name: "field-label",
     label: "Field Label",
     rowValueDescriptor: "label_short",
-    formatter: (x: any, isRow: boolean, field: ILookmlModelExploreField) => {
+    Formatter: function RenderLabelShort({ x, field }: FormatterProps) {
       if (field.field_group_label && field.field_group_variant) {
-        return `${field.field_group_label} ${field.field_group_variant}`
+        return <>{`${field.field_group_label} ${field.field_group_variant}`}</>
       } else {
-        return x
+        return <>{x}</>
```

Comment:
We need a runtime verification that x is a string, since it's typed as `any`

```
  else if (typeof x === 'string') {
    return x
  } else {
    throw new TypeError(...) // will be caught by nearest react error boundary
  }
```